website/blog/2025-12-05-kubernetes-ai-conformance-aks/index.md

```diff
@@ -0,0 +1,180 @@
+
+To create an AI-conformant AKS cluster, be sure to choose a certified Kubernetes version with the appropriate features enabled.
+
+### Step 1. Create an AKS cluster
+
+AKS achieved AI Conformance certification starting with Kubernetes 1.34. Create an AKS cluster running Kubernetes 1.34 or later:
+
+```bash
+az aks create \
+  --resource-group <resource-group> \
+  --name <cluster-name> \
+  --enable-azure-monitor-metrics \
+  --kubernetes-version 1.34.0
+```
+
+### Step 2. Enable the Istio service mesh with Gateway API (optional for advanced routing)
+
+If your AI workloads require advanced traffic management capabilities, you can use the [Istio service mesh add-on with the Gateway API (preview)](https://learn.microsoft.com/azure/aks/istio-gateway-api).
+
+First, install the Azure CLI preview extension and register the feature flag:
+
+```bash
+# Install the aks-preview extension
+az extension add --name aks-preview
+
+# Register the Gateway API feature flag
+az feature register --namespace "Microsoft.ContainerService" --name "ManagedGatewayAPIPreview"
+```
+
+Then enable the Istio service mesh and Gateway API on your cluster:
+
+```bash
+az aks mesh enable \
+  --resource-group <resource-group> \
+  --name <cluster-name>
+
+az aks update \
+  --resource-group <resource-group> \
+  --name <cluster-name> \
+  --enable-gateway-api
+```
+
+### Step 3. Add a GPU node pool
+
+For GPU-accelerated workloads, add a [fully managed GPU node pool (preview)](https://learn.microsoft.com/azure/aks/aks-managed-gpu-nodes) with [GPU-enabled VMs](https://learn.microsoft.com/azure/virtual-machines/sizes/overview#gpu-accelerated). For a select set of NVIDIA GPU SKUs, AKS automatically installs the GPU driver, device plugin, and DCGM metrics exporter.
+
+First, register the feature flag:
+
+```bash
+az feature register --namespace Microsoft.ContainerService --name ManagedGPUExperiencePreview
+```
+
+Then add the GPU node pool with the `EnableManagedGPUExperience` tag:
+
+```bash
+az aks nodepool add \
+  --resource-group <resource-group> \
+  --cluster-name <cluster-name> \
+  --name gpunp \
+  --node-count 1 \
+  --node-vm-size Standard_NC40ads_H100_v5 \
+  --node-taints sku=gpu:NoSchedule \
+  --enable-cluster-autoscaler \
+  --min-count 1 \
+  --max-count 3 \
+  --tags EnableManagedGPUExperience=true
+```
+
+Your cluster now meets the requirements for Kubernetes AI Conformance. You can now optionally [install and run Kueue](https://learn.microsoft.com/azure/aks/kueue-overview) to enable gang scheduling and [deploy batch jobs](https://learn.microsoft.com/azure/aks/deploy-batch-jobs-with-kueue).
+
+## Get involved
+
+The Kubernetes AI Conformance Program is developed in the open. You can contribute by:
+
+- Joining the [WG AI Conformance](https://github.com/kubernetes/community/tree/master/wg-ai-conformance) working group
+- Participating in discussions on the [Kubernetes Slack #wg-ai-conformance channel](https://kubernetes.slack.com/messages/wg-ai-conformance)

```

Comment:
Please remove the above 2 lines about wg AI conformance since that will be moved from a working group to a subproject. Let's remove anything here that makes this look out of date. https://github.com/cncf/k8s-ai-conformance referenced below will have more up to date info on how people can participate.